server/db/migrate/20190528034230_update_block_reward.rb

```diff
@@ -0,0 +1,8 @@
+class UpdateBlockReward < ActiveRecord::Migration[5.2]
+  def up
+    current_epoch_number = api.get_current_epoch.number.to_i
```

Comment:
does `current_epoch_number` can find from database?

Reply:
yes, Thank you.